### PR TITLE
Update rules and steps to contribute in the README

### PR DESCRIPTION
**Pull request by adiati98**
## Description

This PR updates the rules and steps to contribute in the README with details as follow:

- Added a rule for accepting only one added item per contributor.
- Updated the steps to contribute.
  - Added a step to create a new branch.
  - Improved the wordings in step 4.
  - Added links for the GitHub and git commands.

This PR also updates:
- The rule to use dash (-) instead of asterisks (*) for unordered list in the `style-guide.md`.
- The unordered list in `pizza-facts-&-trivia.md` and `regional-pizza.md` by replacing asterisks with dashes.

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

Relates to #31 

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [x] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

there's still one more left GIF

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,34 +4,48 @@
...
-4. Follow our `style-guide.md` to format your content.
-5. Once you're done, **commit** your changes and **push** them to your forked repository.
-6. Submit a **pull request** to have your contribution reviewed and considered for merging into the main repository.
-7. Engage with other contributors and maintainers through comments and discussions.
+1. [Fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo#forking-a-repository) this repository to your own GitHub account.
+2. [Clone](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository#cloning-a-repository) the forked repository to your local machine.
+3. [Create a new branch](https://www.shellhacks.com/git-create-new-branch-and-checkout/) with a name related to your contribution, for example, `feature/add-pizza-recipe`.
+4. You can now add a new pizza fact or trivia to the `pizza-facts-&-trivia.md`, a new regional pizza to the `regional-pizza.md`, or a new recipe to the `pizza-recipes.md` file. Remember, you may only submit one new item in a file.
+5. Follow our `style-guide.md` to format your content.
```
Comment (CBID2) [inline README.md:45]: [suggestion] hyperlink the guide so contributors have quicker access to it :) 
```suggestion
5. Follow [our `style-guide.md`](https://github.com/open-sauced/pizza-verse/blob/main/style-guide.md)
to format your content.
```

File: README.md [hunk 2]
```diff
@@ -4,34 +4,48 @@
...
-3. Add a new pizza fact to the `pizza-facts.md` file or choose to add a new recipe to the `pizza-recipes.md` file.
-4. Follow our `style-guide.md` to format your content.
-5. Once you're done, **commit** your changes and **push** them to your forked repository.
-6. Submit a **pull request** to have your contribution reviewed and considered for merging into the main repository.
-7. Engage with other contributors and maintainers through comments and discussions.
+1. [Fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo#forking-a-repository) this repository to your own GitHub account.
+2. [Clone](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository#cloning-a-repository) the forked repository to your local machine.
+3. [Create a new branch](https://www.shellhacks.com/git-create-new-branch-and-checkout/) with a name related to your contribution, for example, `feature/add-pizza-recipe`.
+4. You can now add a new pizza fact or trivia to the `pizza-facts-&-trivia.md`, a new regional pizza to the `regional-pizza.md`, or a new recipe to the `pizza-recipes.md` file. Remember, you may only submit one new item in a file.
```
Comment (CBID2) [inline README.md:44]:
```suggestion
4. You can now add a new pizza fact or trivia to the [`pizza-facts-&-trivia.md`](https://github.com/open-sauced/pizza-verse/blob/main/pizza-facts-%26-trivia.md),
a new style of pizza to the [`regional-pizza.md`](https://github.com/open-sauced/pizza-verse/blob/main/regional-pizza.md), or a new recipe to the [`pizza-recipes.md`](https://github.com/open-sauced/pizza-verse/blob/main/pizza-recipes.md) file. Remember, you may only submit **one** new item in a file.
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @adiati98. You're off to a great start! 😊 I just gave some tips on how to tweak your PR.

**Comment by adiati98** [comment]
@CBID2 done 😊

**Review by CBID2 — APPROVED** [review]
Looks great @adiati98! 😊 Merge time

**Pull request merged**